test/report_convert_to_pdf_test.py

```diff
@@ -0,0 +1,27 @@
+from os import remove, listdir
+from os.path import exists
+
+import pytest
+from docx import Document
+
+from src.report.report_utils import convert_docx_to_pdf
+
+
+@pytest.fixture(autouse=True)
+def file_cleanup():
+    yield
+    for file in listdir('./'):
+        if file.endswith('.docx'):
+            remove(file)
+        if file.endswith('.pdf'):
+            remove(file)
+
+
+@pytest.mark.skip(reason="Shouldn't be run automatically")
```

Comment:
Why this shouldn't be run automatically?

Reply:
Added libreoffice installation to testing workflow and enabled this test